packages/flutter_tools/lib/src/features.dart

```diff
@@ -54,6 +54,9 @@ abstract class FeatureFlags {
   /// Whether the CFE experimental invalidation strategy is enabled.
   bool get isExperimentalInvalidationStrategyEnabled => true;
 
+  /// Whether the windows UWP embedding is enabled.
+  bool get isWindowsUWPEnabled => false;
```

Comment:
Very nitpicky and I bet we're inconsistent about this, and also have lots of exceptions to Effective Dart in our code, but... generally use CamelCase for abbreviations/initialisms longer than two characters. e.g. `Url`, `HttpClient`.


https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words

Reply:
Fixed. In my defense, this one is hard to follow :)